crates/bevy_app/src/main_schedule.rs

```diff
@@ -352,3 +366,38 @@ impl FixedMain {
         });
     }
 }
+
+/// Set enum for the systems that want to run inside [`RunFixedMainLoop`],
+/// but before or after the fixed update logic. Systems in this set
+/// will run exactly once per frame, regardless of the number of fixed updates.
+/// They will also run under a variable timestep.
+///
+/// This is useful for handling things that need to run every frame, but
+/// also need to be read by the fixed update logic. A good example of this
+/// is camera movement, which needs to be updated in a variable timestep,
+/// as you want the camera to move with as much precision and updates as
+/// the frame rate allows. A physics system that needs to read the camera
+/// position and orientation, however, should run in the fixed update logic,
+/// as it needs to be deterministic and run at a fixed rate for better stability.
+/// Note that we are not placing the camera movement system in `Update`, as that
+/// would mean that the physics system already ran at that point.
+///
+/// # Example
+/// ```
+/// # use bevy_ecs::prelude::*;
+/// App::new()
+///   .add_systems(
+///     RunFixedMainLoop,
+///     update_camera_rotation.in_set(AroundFixedMainLoopSystem::Before))
+///   .add_systems(FixedMain, update_physics);
+///
+/// # fn update_camera_rotation() {}
+/// # fn update_physics() {}
+/// ```
+#[derive(Debug, Hash, PartialEq, Eq, Clone, SystemSet)]
+pub enum AroundFixedMainLoopSystem {
```

Comment:
Actually, a smarter idea: add a third variant for `run_fixed_main_loop` itself!

Reply:
I considered it, but wouldn't you realistically want to schedule things inside `FixedUpdate` and friends instead? Adding this variant seems a bit like a footgun.

Reply:
It would be nice for any logic that's "I want it to run after all the other before things but before `FixedMain`", without needing to mention the specific system (which is also easy to get wrong since there's two systems with roughly the same name

Reply:
@NiseVoid ah, that's a good point, thanks! I'll implement that later then.